sdk/storage/inc/blobs/internal/protocol/blob_rest_client.hpp

```diff
@@ -2059,25 +2078,124 @@ namespace Azure { namespace Storage { namespace Blobs {
         return SetMetadataParseResponse(context, std::move(pResponse));
       }
 
-      struct ListBlobsOptions
+      struct ListBlobsFlatOptions
+      {
+        Azure::Core::Nullable<int32_t> Timeout;
+        Azure::Core::Nullable<std::string> Prefix;
+        Azure::Core::Nullable<std::string> Marker;
+        Azure::Core::Nullable<int32_t> MaxResults;
+        ListBlobsIncludeItem Include = ListBlobsIncludeItem::None;
+      }; // struct ListBlobsFlatOptions
+
+      static Azure::Core::Http::Request ListBlobsFlatConstructRequest(
+          const std::string& url,
+          BodyStreamPointer& body,
+          const ListBlobsFlatOptions& options)
+      {
+        unused(body);
+        auto request = Azure::Core::Http::Request(Azure::Core::Http::HttpMethod::Get, url);
+        request.AddHeader("Content-Length", "0");
```

Comment:
Do we have an issue tracking this?

Reply:
what's wrong with this line of code?

Reply:
Why do we need to explicitly add `"Content-Length: 0"` to each request? And even for `GET` request?
I know we are working around a bug, but I was asking if we have an issue tracking this bug.

Reply:
https://github.com/Azure/azure-sdk-for-cpp/issues/307

Reply:
So why don't you clarify what this issue is at the first comment?